dts/bindings/serial/st,stm32-lpuart.yaml

```diff
@@ -29,4 +29,10 @@ properties:
       category: required
       description: Clock gate information
       generation: define
+
+    hw-flow-control:
+      type: boolean
+      category: optional
+      description: use RTS/CTS flow control
```

Comment:
This might be misleading to people not used to dts (even if already used in other bindings), this is not a boolean that expect a true/false value. Either it is present and means enabled or it is not there andthen flow control is disabled. "Boolean" applies to the generated  flag, not to the node property.
What about: when set, RTC/CTS flow control is enable at boot time.

Reply:
Yep, fits